zk_toolbox/crates/common/src/prerequisites.rs

```diff
@@ -11,10 +11,6 @@ const PREREQUISITES: [Prerequisite; 6] = [
         name: "docker",
         download_link: "https://docs.docker.com/get-docker/",
```

Comment:
should we check the version then? I do remember we had a lot of trouble with it a year ago